backend/auction/views.py

```diff
@@ -317,7 +317,7 @@ def get(self, request, **kwargs):
 
 
 class AuctionItemsApiView(APIView):
-    permission_classes = [IsAuthenticated]
+    permission_classes = [AllowAny]
```

Comment:
@kevinrczhang Any reason why we made is AllowAny?

Reply:
whoops forgot to delete that